gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
```

Comment:
IntelliJ recommends using the `-all` distribution and I personally recommend using the `distributionSha256Sum` parameter to just make sure that you don't pull a manipulated Gradle distribution.

Reply:
The "all" distribution is 40MB bigger, what benefit do you see in pulling it instead of bin? Regarding distributionSha256Sum, I wasn't aware this feature existed, probably doesn't hurt, but also sounds a bit paranoid.

Reply:
It was just a recommendation by IntelliJ. I think, it was typically something regarding debugging IIRC, but I am not fully sure anymore.

Reply:
let's start with the bin. Can later be changed in case we find it useful.